draft-ietf-lamps-kyber-certificates.md

```diff
@@ -409,6 +409,14 @@ ensure that the sender properly generated the private key.
 If the check is done and the seed and the expandedKey are not consistent,
 the recipient MUST reject the private key as malformed.
 
+When receiving a private key that contains an `expandedKey`,
+{{FIPS203}} stipulates in section 7.3 that before use,
+a "hash check" MUST be performed. This section stipulates two other
```

Comment:
Where are two other checks stipulated? In particular what is the "type" check?

Reply:
@bwesterb I think the two extra checks are ASN.1 checks/constraints for type and length.

Reply:
@vdukhovni wrote

> Where are two other checks stipulated? 

Let me copy FIPS 203 section 7.3 for you:

<img width="802" alt="Screenshot 2025-04-11 at 12 13 17" src="https://github.com/user-attachments/assets/26c13471-c9c5-4248-9cb8-477525fb4a60" />

The "decapsulation key type check" is already ensured by this I-D.